src/js/apiData.js

```diff
@@ -1,5 +1,136 @@
-const getApiData = () => {
-  console.log("api data are ready");
-};
+/* 
+  W dużym skrócie metoda getApiData zwraca 6 elementowa tablice [każdy element odpowiada 1 dniowi], każdy element jest obiektem który zawiera informacje odnośnie pogody,
+  każdy element ma takie pola :
+    date - data w postaci stringa,
+    temp - temperatura(w prognozach na nastepne 5 dni jest to srednia danych z całego dnia(8 pomiarów zazwyczaj))
+    temp_min - temperatura minimalna (w prognozach jest to minimalna temperatura ze wszystkich minimalnych temperatur pomiarów)
+    temp_max - ta samo co wyżej
+    weather - pogoda opisana za pomoca stringa (np. "Clouds") - w prognozach podałem to co wystepuje najczesciej ale to mozna ewentualnie zmienic
+    desciption - nie wiem czy ta informacja była potrzebna jest to rozszerzenie informacji weather, rowniez brałem najczęsciej występującą
+    icon - tutaj też nie wiedziałem czy się przyda ale jak coś jest
+    wind_speed - prędkość wiatru (w prognozach średnia z całego dnia zaokrąglona)
+    pressure -  ciśnienie (w prognozach jak wyżej)
+    humidity - wilgotność (również)
+  W razie niepewności albo błędów/niedopatrzeń z mojej strony z chęcią przyjmę krytykę i postaram się to naprawić
+*/
+
+
+async function getApiData(cityName){
```

Comment:
api data to tez zbyt malo precyzyjne okreslenie. co jesli bedziemy miec w projekcie 2 rozne api?